### PR TITLE
[#263] Add storing of model metrics data separately for each edpoint

### LINKED ISSUE #263: Store model metrics separately for each endpoint

### PR DESCRIPTION
**Pull request by skayerok**
Closes #263

### CODE REVIEW

**Comment by dsuslov** [comment]
bot, test all

File: tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot [hunk 1]
```diff
@@ -29,6 +30,11 @@ Checking property update callback
     ${edge}=        Build enclave EDGE URL  ${MODEL_TEST_ENCLAVE}
                     Get token from EDI      ${MODEL_TEST_ENCLAVE}   ${model_id}    ${model_version}
 
+    Log   Connecting to Grafana
+    Connect to enclave Grafana  ${MODEL_TEST_ENCLAVE}
+    Sleep   15s
```
Comment (kirillmakhonin) [inline tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot]: Please do not use `sleep` - use retires instead

File: tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot [hunk 2]
```diff
@@ -37,6 +43,9 @@ Checking property update callback
     Ensure model property has been updated  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_PROP}  1
     Ensure model API call result field is correct  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_ENDPOINT}  result  30   a=1  b=2
 
+    Sleep   15s
```
Comment (kirillmakhonin) [inline tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot]: Same

File: legion_test/legion_test/robot/grafana.py [hunk 3]
```diff
@@ -135,3 +138,54 @@ def metric_should_be_presented(self, model_id, model_version):
                 break
         else:
             raise Exception('Cannot find any value > 0')
+
+    def metric_should_not_be_presented(self, model_id, model_version, model_endpoint='default'):
+        """
+        Check that requests count metric for model exists
```
Comment (kirillmakhonin) [inline legion_test/legion_test/robot/grafana.py]: Wrong description

File: legion_test/legion_test/robot/grafana.py [hunk 4]
```diff
@@ -135,3 +138,54 @@ def metric_should_be_presented(self, model_id, model_version):
                 break
         else:
             raise Exception('Cannot find any value > 0')
+
+    def metric_should_not_be_presented(self, model_id, model_version, model_endpoint='default'):
```
Comment (kirillmakhonin) [inline legion_test/legion_test/robot/grafana.py]: Duplicated function

Comment (kirillmakhonin) [reply legion_test/legion_test/robot/grafana.py]: Sorry, not the function is duplicated,but the main part of function

**Comment by skayerok** [comment]
bot, test all

**Comment by kirillmakhonin** [comment]
bot, test all

**Comment by skayerok** [comment]
bot, test all

**Comment by kirillmakhonin** [comment]
bot, test all

**Comment by skayerok** [comment]
bot, test all

**Comment by skayerok** [comment]
bot, test all

File: tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot [hunk 5]
```diff
@@ -37,6 +42,8 @@ Checking property update callback
     Ensure model property has been updated  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_PROP}  1
     Ensure model API call result field is correct  ${model_id}  ${model_version}  ${edge}  ${TOKEN}  ${MODEL_WITH_PROPS_ENDPOINT}  result  30   a=1  b=2
 
+    Ensure metric present  ${model_id}  ${model_version}  ${MODEL_WITH_PROPS_ENDPOINT}
```
Comment (skayerok) [inline tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot]: > 
> 
> Maybe we have to add tests for non-default endpoint? What do you think?

I have already added a case for non-default endpoint. Is that OK?

Comment (kirillmakhonin) [reply tests/robot/tests/2_jenkins_models/2.0_check_jenkins_models.robot]: OK, that's good

**Pull request merged**